contrib/create-static-manifest.sh

```diff
@@ -25,7 +25,7 @@ for filepath in $TEMPLATE_FILE; do
     filename=$(basename $filepath)
     outputname="${filename%%.*}.yml"
     # Use helm to generate the yaml and then use sed to remove the helm specific lables/annotations.
-    helm template "$CHART_DIR" --name=openfaas --namespace="$NAMEPSPACE" --set "functionNamespace=$FUNCTIONNAMESPACE" -x templates/$filename --values="$VALUESNAME" \
+    helm template openfaas "$CHART_DIR" --namespace="$NAMEPSPACE" --set "functionNamespace=$FUNCTIONNAMESPACE" -s templates/$filename --values="$VALUESNAME" \
```

Comment:
Is that a change from helm2 to helm3 or something else?

Reply:
i think so, it simply wouldn't work with helm3

Reply:
I kept on using helm2 for this which arkade can download, but will make a note that this project needs helm3 to generate charts. now